packages/php-wasm/compile/php/php_wasm.c

```diff
@@ -1026,8 +1026,8 @@ int main(int argc, char *argv[]);
 int run_cli()
 {
 	// See wasm_sapi_request_init() for details on why we need to redirect stdout and stderr.
-	stdout_replacement = redirect_stream_to_file(stdout, "/internal/stdout");
-	stderr_replacement = redirect_stream_to_file(stderr, "/internal/stderr");
+	stdout_replacement = redirect_stream_to_file(stdout, "/request/stdout");
```

Comment:
I like that new path

Reply:
I'm glad to hear it! I like it too but wasn't sure what you'd think.